packages/contracts-bedrock/test/L1/L1CrossDomainMessenger.t.sol

```diff
@@ -631,6 +632,8 @@ contract L1CrossDomainMessenger_ReinitReentryTest is Bridge_Initializer {
             l1CrossDomainMessenger.initialize(SuperchainConfig(superchainConfig));
 
             // attempt to re-replay the withdrawal
+            vm.expectEmit(address(l1CrossDomainMessenger));
+            emit FailedRelayedMessage(hash);
```

Comment:
Addresses [this request](https://github.com/ethereum-optimism/optimism/pull/8797#discussion_r1442353495): 
> Could we add an expect emit here to clarify what codepath this is supposed to take?